go.mod

```diff
@@ -1,3 +1,14 @@
 module billion-rows
 
-go 1.24
+go 1.24.6
+
+require (
+	github.com/stretchr/testify v1.10.0
```

Comment:
hmm I wouldn't pull all of testify for this though I guess tests are very good to have and you wrote them with testify already so 🎉